chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -320,9 +320,17 @@ pub(crate) fn pre_validate_chunk_state_witness(
 
     let main_transition_params = if last_chunk_block.header().is_genesis() {
         let epoch_id = last_chunk_block.header().epoch_id();
+        let congestion_info = last_chunk_block
+            .shards_congestion_info()
+            .get(&shard_id)
+            .map(|info| info.congestion_info);
```

Comment:
nit: This is often repeated (also outside of this PR) - would it make sense to refactor that to a method?

Reply:
I can take a look to do the refactor separately, yes. But form a brief code search, I couldn't find more than one other place that could potentially share this logic.